### PR TITLE
Update project thread update docs

### LINKED ISSUE #2249: Project Management Improvements
This is a tracking issue to manage some improvements to our project process identified in discussion #1171.

- [ ] Remove the template for project updates. Add examples of more and less-effective project updates in our documentation. Updates will be comments on the project thread which provide a description of changes made to the project.
- [ ] Add documentation to encourage regular updates to the project thread body. This is where document and issue links should be added.
- [ ] Update documentation to indicate any member of the project can and should update the project. If you author and merge a significant part of a project, or make any update to a project that folks should know about, leave a comment in the project thread.
- [x] #2250

### PR DESCRIPTION
**Pull request by zackkrida**
## Fixes

Fixes #2249 by @zackkrida 

## Description

Updates project thread update documentation, removes the template, and links to some real-world examples from the repo. Oh, and also fixes some unrelated typos.

## Testing Instructions

Let me know if you think this is sufficient information for project leads and contributors on how to leave updates. Keep in mind that folks will be directed to this section from the automated reminders in #2251, which state:

> Hi @${recipient}, this project has not received an update comment in 14 days. Please leave an update comment as soon as you can. See the [documentation on project updates](https://docs.openverse.org/projects/planning.html#providing-project-updates) for more information.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [x] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by AetherUnbound — APPROVED** [review]
This looks great, and I appreciate the links to existing examples too!

File: documentation/projects/planning.md [hunk 1]
```diff
@@ -71,57 +71,36 @@ relevant step of the process is complete.
...
-update is "the project is blocked." or "No developments this week.", the updates
-should still be posted. This removes ambiguity for folks visiting the project,
-avoiding questions like "I don't see an update this week; is it being worked on
-or not?"
+For all active projects (anything that isn't in the `Not Started`, `Completed`,
+and `On Hold` statuses), fortnightly public updates are the best way for
+Openverse contributors and maintainers to understand the health of a project.
+Importantly, these updates allow us to allocate more time or material resources
+to projects that need them before work stalls completely. They also allow for
+identifying projects that have stalled and may need special attention, if they
```
Comment (sarayourfriend) [inline documentation/projects/planning.md:79]: Nit: small suggestion to simplify the language

```suggestion
to projects that need them before work stalls completely. They also help identify projects that have stalled and may need special attention, if they
```

File: documentation/projects/planning.md [hunk 2]
```diff
@@ -71,57 +71,36 @@ relevant step of the process is complete.
...
+haven't seen an update in some time.
+
+Project updates should be left as comments in the project threads by the project
+lead or anyone working on a project. If you merge or a significant update to a
+project, or identify a bug that impacts an open project, please leave an update!
```
Comment (sarayourfriend) [inline documentation/projects/planning.md:84]: Looks like a small editing error occurred here, suggested a fix for the typo and slight clarity.

```suggestion
lead or anyone working on a project. If you merge a significant PR related to the project or identify a bug that impacts an open project, please leave an update!
```

File: documentation/projects/planning.md [hunk 3]
```diff
@@ -71,57 +71,36 @@ relevant step of the process is complete.
...
-
-<!-- List any blockers for the week: any upcoming concerns which need to be
-addressed. Please @ anyone who may be qualified to resolve these blockers. The
-more proactive you are about potential solutions the quicker we can make
-eliminate these blockers. -->
-```
+There are automated reminders configured for these updates via a
+[daily GitHub action](https://github.com/WordPress/openverse/blob/main/.github/workflows/project_thread_update_reminders.yml)
+which comments on the project thread. The project lead is reminded (via a bot
+comment on the project) to provide an update.
```
Comment (sarayourfriend) [inline documentation/projects/planning.md:103]: Nit for clarity/splificiation.

```suggestion
that comments on the project thread, prompting the project lead to provide an update.
```

**Review by sarayourfriend — APPROVED** [review]
I've left some small editorial suggestions but otherwise this LGTM and captures the result of the discussion well.

**Pull request merged**